README.md

```diff
@@ -1 +1,47 @@
-# Busify
\ No newline at end of file
+<p align-self="center"><img src="./assets/images/Logo.gif" alt="Alt Text"></p>
```

Comment:
The image looks a little ugly here. We can remove this image for now. We could make something custom for here later.